### PR TITLE
new post

### PR DESCRIPTION
**Pull request by freeman-lab**
PR to add a new post. Also includes a `Blockquote` component to do hanging indents for quotation marks inside blockquotes. We may want to eventually move this to `@carbonplan/components` or similar.

### CODE REVIEW

File: components/blockquote.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Box } from 'theme-ui'
+
+const Blockquote = ({ children }) => {
+  const firstChar = children.props.children.slice(0, 1)
+  const textIndent = firstChar === '“' ? ['-0.4em'] : [0]
```
Comment (katamartin) [inline components/blockquote.js:5]: nit: should we also check against `'"'`?

File: components/blockquote.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <Box
+      sx={{
+        bg: 'hinted',
```
Comment (katamartin) [inline components/blockquote.js:9]: Is it possible to use `variant: 'styles.blockquote'` here instead of manually specifying the `blockquote` styles?

**Review by katamartin — COMMENTED** [review]
Two minor comments on the `Blockquote` component -- lgtm otherwise!

File: components/blockquote.js (hunk 1 above)
Comment (freeman-lab) [reply components/blockquote.js:5]: haha welllll ideally we shouldn't use that to indicate the start of a quotation (and it'll have an ever so slightly different width), so maybe no?

**Pull request merged**